src/clr-angular/forms/select/option.ts

```diff
@@ -3,13 +3,26 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import {Component, ElementRef, HostListener, Inject, Optional} from "@angular/core";
+import {Component, ElementRef, HostBinding, HostListener, Inject, Optional} from "@angular/core";
 import {POPOVER_HOST_ANCHOR} from "../../popover/common/popover-host-anchor.token";
 import {IfOpenService} from "../../utils/conditional/if-open.service";
+import {OptionSelectionService} from "./providers/option-selection.service";
 
 @Component({selector: "clr-option", templateUrl: "option.html", host: {"[class.clr-option]": "true"}})
 export class ClrOption {
-    constructor(private ifOpenService: IfOpenService, @Optional() @Inject(POPOVER_HOST_ANCHOR) parentHost: ElementRef) {
+    private _selected: boolean = false;
+
+    @HostBinding("class.active")
+    get selected(): boolean {
```

Comment:
You have a getter and a setter that don't do anything special, we avoid this to keep things simple. Just use a public property:
```typescript
@HostBinding("class.active") selected = false;
```